### ISSUE TITLE
Humidity

### ISSUE THREAD

**Issue by mariusnauj** [MichalKrasowski/tech-controllers#15]
I have room termostats with humidity data and I see this info in Tech application or web. It's possible to get this data and i HA ?

**Comment by jontofront** [comment]
Which models do you have?

**Comment by mariusnauj** [comment]
I have this:

https://tech-controllers.com/p/eu-r-8b-plus

They have Humidity info, and I see this info in each zone in Tech controller app or web.

**Comment by jontofront** [comment]
Yes, you can it's my example
![image](https://user-images.githubusercontent.com/37038426/200837342-5aeae3f0-33f4-4e8b-9ee9-f6ee2179304d.png)

Also you cn extract it separatly. example:
`
- platform: template
  sensors:
    temperature:
      unique_id: living_room_current_temp
      friendly_name: "Living room current temp"
      device_class: "temperature"
      unit_of_measurement: "°c"
      value_template: "{{ state_attr('climate.living_room', 'current_temperature') }}"
`

**Comment by mariusnauj** [comment]
Yes, but i'm speaking about Humidity not temperature. In this integration with HA I dont have this data.
